docs/angular/guides/modern-angular/protractor-to-cypress.md

```diff
@@ -0,0 +1,41 @@
+# Migrating from Protractor to Cypress
+
+Nx helps configure your e2e tests for you. You can choose Protractor as an option, but the default is Cypress. If you have an existing set of e2e tests using Protractor and would like to switch to using Cypress, you can follow these steps.
+
+Let's say your existing app is named `my-awesome-app` and the e2e Protractor tests are located in `my-awesome-app-e2e`.
+
```

Comment:
Maybe as a step 0 or a note to make a git commit - or start on a fresh branch - before doing these steps